src/content/docs/cypher/query-clauses/call.md

```diff
@@ -22,6 +26,13 @@ The following tables lists the built-in schema functions you can use with the `C
 | `SHOW_OFFICIAL_EXTENSIONS` | returns all official [extensions](/extensions) which can be installed by `INSTALL <extension_name>` |
 | `SHOW_LOADED_EXTENSIONS` | returns all loaded extensions |
 | `SHOW_INDEXES` | returns all indexes built in the system |
+| `SHOW_PROJECTED_GRAPH` | returns all projected graph created in the system |
```

Comment:
I think this function name should be `SHOW_PROJECTED_GRAPHS`. I will open an PR for the renaming on main repo.